### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by pileofkyle** [Dilbert66/esphome-dsckeybus#107] [General]
This is so awesome. Thanks for putting this all together. Love the virtual expansion mods and triggers! endless fun.

**Comment by fctruter** [comment]
Just want to echo this comment, thanks so much for an amazing project, I appreciate all of your hard work!

**Comment by Dilbert66** [comment]
Glad you folks like the project! If you have feature requests, just ask and I can investigate it.
